### PR TITLE
1.0 refactor

### PR DESCRIPTION
**Pull request by aserv92**
Refactored my changes to flexible mink into small functions without boolean voodoo.

### CODE REVIEW

**Comment by aserv92** [comment]
!force code review

**Comment by healthlabs-bugman** [comment]
Paging @Chekote to review the back-end work in this pull request!
Paging @awesomeousPrime to review the testing work in this pull request!

**Comment by aserv92** [comment]
Reassigned reviewers to match 2.0 refactor that has already been merged.
PR#211

File: src/Behat/FlexibleMink/Models/Geometry/Rectangle.php [hunk 1]
```diff
@@ -7,74 +7,47 @@
  */
 class Rectangle
```
Comment (PrinceMegahit) [inline src/Behat/FlexibleMink/Models/Geometry/Rectangle.php]: Usually `Rectangle` classes are implemented as an X,Y `Point` tuple and a width,height `Point` tuple, with properties for accessing right left topleft topright etc. But this is great as it stands.

**Pull request merged**